### ISSUE TITLE
CSS adjustments

### ISSUE THREAD

**Issue by jvine** [sul-dlss/library_hours_rails#15]
- [x] remove alt-row striping from tables
- [x] remove letterspacing from h1 (only necessary in SW with dejavu font)
- [x] Stanford btn-default styling seems to be missing (default is grey instead of beige)
- [x] set base font size to 15px
- [x] set base line-height to 1.5em
- [x] currently all dates are showing as exceptions - either kill the style or make it exceptional 😄 
- [x] new colours for open24h/closed:
```
.open-24h {color: #009b76;}
.closed {color: #dd2121;}
```
- [x] link styling:
```
a {color: #A26624; text-decoration: none; border-bottom: 0;}
a:active, a:hover {color: #A26624; border-bottom: 2px solid #A26624;}
a:focus {outline: 5px auto #eaab00 !important; border-bottom: 2px solid #A26624; outline-offset: 0;}
```

Also note the heading styles in #13

**Comment by cbeer** [comment]
> all dates are showing as exceptions 

Only because we haven't convinced access services staff to set normal hours for their locations, but the current process is wedded to nasty spreadsheets getting passed around. The intent was to make it easy for staff to see where non-standard hours were set.

**Comment by cbeer** [comment]
Fixed by #17
